src/pages/login-page/login-page.tsx

```diff
@@ -22,14 +22,20 @@ export const LoginPage = observer(() => {
     reset,
   } = useForm<LoginFormValues>();
 
-  const onSubmit = (data: LoginFormValues) => {
-    void userStore.login(data);
+  const onSubmit = async (data: LoginFormValues) => {
+    const customer = {
+      email: data.email,
+      password: data.password,
+    };
```

Comment:
там точно такие же поля
```suggestion
  const onSubmit = async (customer: LoginFormValues) => {
```

Reply:
ну или data назвать и ее передавать userStore.login(data). суть в том чтоб не перекладывать из одного объекта в такой же